voa/copula.py

```diff
@@ -26,23 +28,25 @@ def create_Q_grid(n=10):
 
     return grid
 
+
 @numba.njit(parallel=True)
 def calculate_copula_part(t, c):
```

Comment:
Please type function signature, and use more verbose parameter names

Reply:
I added the signatures. The names are kept according to the paper and the initial implementation. We can change it, but I think it will not make the code more readable as someone will need to "translate" the names from our code to the equations from original c# implementation.